Languages/IronPython/IronPythonCompiler/Program.cs

```diff
@@ -45,7 +45,7 @@ class Program {
             var u = new Universe();
             var aName = new AssemblyName(Path.GetFileNameWithoutExtension(new FileInfo(config.Output).Name));
             var ab = u.DefineDynamicAssembly(aName, AssemblyBuilderAccess.Save, Path.GetDirectoryName(config.Output));
-            var mb = ab.DefineDynamicModule(config.Output, aName.Name + (aName.Name.EndsWith(".exe") ? string.Empty : ".exe"));
+            var mb = ab.DefineDynamicModule(config.Output, Path.ChangeExtension(aName.Name, ".exe"));
```

Comment:
This changes the semantics if the path already has an extension which is not ".exe", is that intentional?

On example would be names with versions - foo3.5 used to be foo3.5.exe and is now foo3.exe.